### PR TITLE
docs: Update Documentation and feat: Added Alert Component

### PR DESCRIPTION
**Pull request by munjalSaksham71**

### CODE REVIEW

File: utils/alert.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    padding: 1rem 0.75rem;
+    margin-bottom: 1rem;
+}
+
+.primary-success {
+    background-color: var(--light-success);
+    color: var(--success);
+    padding: 1rem 0.75rem;
+    margin-bottom: 1rem;
+}
```
Comment (rkapoor10) [inline utils/alert.css:31]: you may have a common class for margin and padding here...also variables for them can also be used.

**Review by rkapoor10 — APPROVED** [review]

**Review by sks-21 — APPROVED** [review]
The code looks nice.
Just one suggestion, the quality of the image on the preview doesn't look good.

File: utils/alert.css (hunk 1 above)
Comment (munjalSaksham71) [reply utils/alert.css:31]: Yes I have - I will do this now. - Thank You!

**Comment by munjalSaksham71** [comment]
> The code looks nice. Just one suggestion, the quality of the image on the preview doesn't look good.

I will try to use other images with different PR

File: docs/docs.html [hunk 2]
```diff
@@ -78,6 +79,40 @@ <h1 class="heading">REGAL UI</h1>
             class="code-snippet"
             sandbox="allow-scripts allow-same-origin">
             </iframe> <hr class="mt-4" />
+
+            <!-- ######################   Alerts  ######################-->
+
+            <div id="alert" class="heading1 bolder mt-5 mb-5">Alert</div>
+            <div class="overview mb-3">An alert displays a short, important message in a way that attracts the user's attention without interrupting the user's task.</div>
+            <!-- Normal Alert -->
+            <div class="primary-alert">This is an primary alert — <span class="bolder">check it out!</span></div>
```
Comment (Priya-Patel31) [inline docs/docs.html]: can improve indentation !!

File: docs/docs.html [hunk 3]
```diff
@@ -78,6 +79,40 @@ <h1 class="heading">REGAL UI</h1>
...
+            class="code-snippet"
```
Comment (Priya-Patel31) [inline docs/docs.html:115]: add margin top to provide spacing between heading and code snippet !!

**Review by Priya-Patel31 — CHANGES_REQUESTED** [review]

File: docs/docs.html (hunk 3 above)
Comment (munjalSaksham71) [reply docs/docs.html:115]: Okk

**Pull request merged**